app/views/subscription_mailer/subscription.html.erb

```diff
@@ -3,6 +3,10 @@
   <% if @subscription.name.present? %>
     "<%= @subscription.name %>"
   <% end %>
+
+  <% if @questions.count == 25 %>
+    This e-mail only shows the 25 most recent items matching your filter.
+  <% end %>
```

Comment:
Probably want to copy this to subscription.text.erb too.

Reply:
Added in #881